Code/Framework/AzToolsFramework/AzToolsFramework/AssetBrowser/Views/AssetBrowserThumbnailView.cpp

```diff
@@ -71,6 +76,82 @@ namespace AzToolsFramework
+                        const AssetBrowserEntry* entry = index.data(AssetBrowserModel::Roles::EntryRole).value<const AssetBrowserEntry*>();
+                        AZStd::vector<const AssetBrowserEntry*> entries{ entry };
+                        AssetBrowserInteractionNotificationBus::Broadcast(
+                            &AssetBrowserInteractionNotificationBus::Events::AddContextMenuActions, this, &menu, entries);
+
+                        if (!menu.isEmpty())
+                        {
+                            menu.exec(QCursor::pos());
+                        }
+                    }
+                    else if (!index.isValid() && m_assetTreeView)
+                    {
+                        m_assetTreeView->OnContextMenu(QCursor::pos());
+                    }
+                });
+
+            connect(m_thumbnailViewWidget, &AzQtComponents::AssetFolderThumbnailView::afterRename, this, &AssetBrowserThumbnailView::AfterRename);
+
+              if (AzToolsFramework::IsNewActionManagerEnabled())
+              {
+                  if (auto hotKeyManagerInterface = AZ::Interface<AzToolsFramework::HotKeyManagerInterface>::Get())
+                  {
+                      // Assign this widget to the Editor Asset Browser Action Context.
+                      hotKeyManagerInterface->AssignWidgetToActionContext(
+                          EditorIdentifiers::EditorAssetBrowserActionContextIdentifier, this);
+                  }
+              }
+
+              QAction* deleteAction = new QAction("Delete Action", this);
+              deleteAction->setShortcut(QKeySequence::Delete);
+              deleteAction->setShortcutContext(Qt::WidgetWithChildrenShortcut);
+              connect(
+                  deleteAction,
+                  &QAction::triggered,
+                  this,
+                  [this]()
+                  {
+                      DeleteEntries();
+                  });
+              addAction(deleteAction);
+
+              QAction* renameAction = new QAction("Rename Action", this);
+              renameAction->setShortcut(Qt::Key_F2);
+              renameAction->setShortcutContext(Qt::WidgetWithChildrenShortcut);
+              connect(
+                  renameAction,
+                  &QAction::triggered,
+                  this,
+                  [this]()
+                  {
+                      RenameEntry();
+                  });
+              addAction(renameAction);
+
+              QAction* duplicateAction = new QAction("Duplicate Action", this);
+              duplicateAction->setShortcut(QKeySequence("Ctrl+D"));
+              duplicateAction->setShortcutContext(Qt::WidgetWithChildrenShortcut);
+              connect(
+                  duplicateAction,
+                  &QAction::triggered,
+                  this,
+                  [this]()
+                  {
+                      DuplicateEntries();
+                  });
+              addAction(duplicateAction);
```

Comment:
Would be nice to register these actions using the new action manager, but I understand this is copied from the other views so this could be done in a follow-up PR.